cms/envs/production.py

```diff
@@ -16,6 +17,28 @@
 
 from openedx.core.lib.derived import derive_settings  # pylint: disable=wrong-import-order
 from openedx.core.lib.logsettings import get_logger_config  # pylint: disable=wrong-import-order
+from django.core.exceptions import ImproperlyConfigured  # pylint: disable=wrong-import-order
```

Comment:
we could... put this in the right place* and remove the import order linting exception

* order is python std lib, 3rd party package, local imports.  So put above our local imports

Reply:
Yeah i'll move it up, I thought perhaps there was some reason to it being that low from before, but i'll try just moving it up